src/components/inputtodo.js

```diff
@@ -0,0 +1,45 @@
+/* eslint-disable */
+import React from 'react';
+
+export default class InputTodo extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      title: '',
+    };
+  }
+
+  onChange = (e) => {
+    this.setState({
+      [e.target.name]: e.target.value,
+    });
+  };
+
+  handleSubmit = (e) => {
+    e.preventDefault();
+    if (this.state.title.trim()) {
+      this.props.addTodoProps(this.state.title);
+      this.setState({
+        title: '',
+      });
+    } else {
+      alert('Please write item');
+    }
```

Comment:
Dear, do not use the alert () function in your code as per Microverse Policy. You may use a console.log () function instead.